e2e/test/scenarios/dashboard/x-rays.cy.spec.js

```diff
@@ -261,6 +261,35 @@ describe("scenarios > x-rays", { tags: "@slow" }, () => {
     cy.findByText("463");
   });
 
+  it("should correctly apply breakout in query builder (metabase#14648)", () => {
+    const options = { timeout: 10000 };
+
+    cy.visit(`/auto/dashboard/table/${ORDERS_ID}`);
+
+    getDashboardCards().contains("18,760", options).click();
```

Comment:
Please add `cy.wait` calls here, intercepting BE calls instead of manual timeout

Reply:
I think we could wait for the dashboard response, find the desired card, and wait for the specific dataset response. Wdyt?

Reply:
Do you mean `/automagic-dashboards/table/:id`?
![image](https://github.com/metabase/metabase/assets/31325167/99cb0c07-67f4-4ae9-85a8-f410aa9f8244)

"Waiting for the specific dataset" part might be a bit problematic because I'm seeing a ton of cancelled requests first:
![image](https://github.com/metabase/metabase/assets/31325167/fd407383-5c8f-43cc-a2d4-2d831604f029)

That's my only concern.
But I like the suggested approach in general. Finding a specific card and waiting only for its dataset is the key here.

Reply:
Which is failing.
As I said, the timeout to find the string in a card might be a better approach here.